src/data.js

```diff
@@ -66,6 +66,27 @@ export default [
       'GraphQL',
     ],
   },
+  {
+    name: 'Amarpreet Singh',
+    description:
+      'Frontend Developer. Things i love JS, React, Swift, practicing minimalism and milk tea enthusiast.',
+    url: 'https://www.momcoded.com/uses/',
+    twitter: '@amarmanhala',
+    emoji: '☕',
+    country: '🇨🇦',
+    computer: 'apple',
+    phone: 'iphone',
+    tags: [
+      'Developer',
+      'Frontend',
+      'Learner',
+      'minimalist',
+      'JavaScript',
+      'React',
+      'CSS',
+      'Swift,
```

Comment:
There's a closing single quote missing here, makes the whole file syntactically invalid